include/forcing/Forcing.h

```diff
@@ -42,6 +42,20 @@ struct forcing_params
     }
 };
 
+//AORC Forcing Data Struct
```

Comment:
I had trouble with an accessor to this struct inside of the class, so I moved it outside of the class.